tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+import opentracing_instrumentation.client_hooks
+
+opentracing_instrumentation.client_hooks.install_all_patches()
```

Comment:
Are you sure we want this in conftest?  I think it's safer to use a fixture that restores the patches after the test, e.g. https://github.com/uber/tchannel-python/blob/native-opentracing-integration/tests/test_tracing.py#L85